package.json

```diff
@@ -22,13 +22,19 @@
     "chai-http": "^3.0.0",
     "chalk": "^2.3.0",
     "dotenv": "^4.0.0",
+    "enzyme": "^3.2.0",
+    "enzyme-adapter-react-16": "^1.1.0",
     "express": "^4.16.2",
     "jest": "^21.2.1",
     "morgan": "^1.9.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-redux": "^5.0.6",
     "react-scripts": "^1.0.17",
-    "redis": "^2.8.0"
+    "react-test-renderer": "^16.2.0",
```

Comment:
Do you use this module?

Reply:
@duyoji no error happened on my end. So I will fix this.